cloud/cluster/cluster.go

```diff
@@ -0,0 +1,65 @@
+package cluster
```

Comment:
We'll want service discovery and cluster membership in every binary in Cloud Scoot.

I noticed this when I implemented service discovery within Twitter and it felt weird to have it reference details of the scheduler when it was implementing a common interface. So I think the interface should be binary-agnostic.

Does that make sense?